core/server/apps/amp/index.js

```diff
@@ -1,15 +1,26 @@
-var router           = require('./lib/router'),
+var router              = require('./lib/router'),
     registerAmpHelpers  = require('./lib/helpers'),
+    config              = require('../../config'),
+    ampIsEnabled;
 
-    // Dirty requires
-    config     = require('../../config');
+ampIsEnabled = function ampIsEnabled() {
```

Comment:
I orientated more from the subscribers app, where `labs` is a util, but used in more than one place throughout the codebase:

https://github.com/TryGhost/Ghost/blob/lts/core/server/apps/subscribers/index.js#L74

Don't know, which way is better/cleaner/worth it?